src/frontend/apps/e2e/__tests__/app-drive/create-folder.spec.ts

```diff
@@ -20,7 +20,7 @@ test("Create a folder", async ({ page }) => {
 
   await expect(page.getByText("Drop your files here")).not.toBeVisible();
   await expect(
-    page.getByRole("cell", { name: "My first folder" })
+    page.getByRole("cell", { name: "My first folder", exact: true })
```

Comment:
Why ?

Reply:
Because adding this includes the folder name: https://github.com/suitenumerique/drive/blob/ea4c9ee5637c01d28cd5cb470865b2a8759dce75/src/frontend/apps/drive/src/features/explorer/components/embedded-explorer/EmbeddedExplorerGridActionsCell.tsx#L132-L134

It results having multiple elements matching this `getByRole` call, before only one was found. By default it matches partial text.